package.json

```diff
@@ -64,11 +63,16 @@
     "eslint": "3.6.0",
     "font-awesome": "4.6.3",
     "handlebars": "4.0.5",
+    "imports-loader": "0.6.5",
```

Comment:
`imports-loader` is still being used to provide the `jQuery` global variable to the jquery libs, which I haven't changed in this diff.